stl/src/ppltasks.cpp

```diff
@@ -235,7 +235,7 @@ namespace Concurrency {
             } else {
                 ComCallData callData;
                 ZeroMemory(&callData, sizeof(callData));
-                callData.pUserDefined = reinterpret_cast<void*>(&_Func);
+                callData.pUserDefined = static_cast<void*>(&_Func);

```

Comment:
Let's just allow this to convert implicitly:
```suggestion
                callData.pUserDefined = &_Func;
```